public/javascripts/SVValidate/mobile/mobileValidate.js

```diff
@@ -1,4 +1,4 @@
-$(document).ready(function() {
+$(document).ready(function() {     
```

Comment:
Oh man, so close :joy: Looks like you accidentally also added a few extra spaces to the end of this line when you were removing that other line that was accidentally added. Can you fix this too?

Just make sure to run `git diff` on each file before you add and commit them to make sure there are only changes listed there that you intended to make!